build/teamcity/cockroach/nightlies/microbenchmark_weekly.sh

```diff
@@ -207,6 +207,7 @@ if [ -d "$output_dir/experiment" ] && [ "$(ls -A "$output_dir/experiment")" ] \
     --sheet-desc="$sheet_description" \
     ${influx_token:+--influx-token="$influx_token"} \
     ${influx_host:+--influx-host="$influx_host"} \
+    ${TRIGGERED_BUILD:+--post-issues} \
```

Comment:
Why this is required and cannot be the default setting? Is there any similar flag available for the slack notification?

Reply:
This flag already exists, I am just using it. See [here](https://github.com/ibreakthecloud/cockroach/blob/f71f5f2f4e72e0cbe6f2d94ba97d7abee9776e8b/pkg/cmd/roachprod-microbench/main.go#L119).